go/export.go

```diff
@@ -11,8 +11,8 @@ func (self *VimLParser) Parse(reader *StringReader, filename string) ast.Node {
 	return newAstNode(self.parse(reader), filename)
 }
 
-func (self *ExprParser) Parse() ast.Node {
-	return newAstNode(self.parse(), "")
+func (self *ExprParser) Parse() ast.Expr {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
receiver name should be a reflection of its identity; don't use generic names such as "this" or "self"